odoo/addons/base/views/ir_module_views.xml

```diff
@@ -162,36 +160,34 @@
                   <templates>
                     <t t-name="kanban-menu">
                         <t t-set="installed" t-value="record.state.raw_value == 'installed'"/>
-                        <a type="edit" class="dropdown-item">Module Info</a>
+                        <a type="open" class="dropdown-item">Module Info</a>
                         <a t-if="record.website.raw_value" role="menuitem" class="dropdown-item o-hidden-ios" t-att-href="record.website.raw_value" target="_blank">Learn More</a>
                         <a t-if="installed" name="button_immediate_upgrade" type="object" role="menuitem" class="dropdown-item" groups="base.group_system">Upgrade</a>
                         <a t-if="installed" name="button_uninstall_wizard" type="object" role="menuitem" class="dropdown-item" groups="base.group_system">Uninstall</a>
                     </t>
-                    <t t-name="kanban-box">
-                      <div class="oe_module_vignette">
-                        <div class="oe_module_icon">
-                            <img t-attf-src="#{record.icon.value}" class="w-100" alt="Icon"/>
-                            <span t-if="record.icon_flag" class="oe_module_flag"><t t-out="record.icon_flag.raw_value"/></span>
+                    <t t-name="kanban-card">
+                        <div class="d-flex align-items-center">
+                            <aside style="width: 50px;">
+                                <field name="icon" widget="image_url" class="w-100" alt="Icon"/>
+                                <field t-if="icon_flag" class="position-absolute" name="icon_flag" style="left: 12px; top: calc(50% - 35px); font-size: 27px; font-family: icon; text-shadow: 0px 0px 5px rgba(0, 0, 0, 0.5);" />
```

Comment:
t-if="icon_flag"? What is this?

Reply:
when I said "try to remove the css", I didn't mean "at all cost". If it implies to put the same rules inline, it's definitely not what we want. What I meant was rather:
 - challenge the rules: are they really necessary?
 - can they be replaced by bootstrap utility class?

Reply:
Result of challenging the class:
- Are they really necessary: yes the css rules are necessary here.
- can they be replaced by bootstrap utility class: no only the `position-absolute` class can be used here everything else we need to put inline.

Conclusion: its better to keep the `oe_module_flag` class to avoid the inline style.